Unix/installbuilder/datafiles/SunOS.data

```diff
@@ -74,3 +74,17 @@ while read dest; do
 done
 [ "$error" = yes ] && exit 2
 exit 0
+
+%Preinstall_10
+egrep "^omi:" /etc/group 2>&1 >/dev/null
+if [ $? -ne 0 ]; then
+    echo "Creating omi group ..."
+    groupadd omi
+fi
+
+%Postuninstall_100
```

Comment:
I think SunOS doesn't have an "upgrade" facility, as such. It actually removes and then adds in order to do an upgrade. If so (validate this, @johnkord might remember off the top of his head), this is okay without further conditionals.